js/widgets/help.js

```diff
@@ -362,7 +362,8 @@ function HelpWidget() {
                 block.helpString;
 
             var helpBody = docById("helpBodyDiv");
-                helpBody.style.height = "";
+                helpBody.style.height = "500px";
```

Comment:
Is that the height of the window widget?

Reply:
@walterbender Yes